### ISSUE TITLE
Error creating newspaper processes: No doctypes configured in opac.xml

### ISSUE THREAD

**Issue by matthias-ronge** [kitodo/kitodo-production#3870]
The generation of newspaper processes fails with the error message "No doctypes configured in opac.xml". However, doctypes are no longer defined in the opac.xml, but in the ruleset.

Stack trace:
```
[ERROR] 2020-07-28 14:17:56.780 [http-nio-8080-exec-9] EmptyTask - No doctypes configured in opac.xml
org.kitodo.exceptions.DoctypeMissingException: No doctypes configured in opac.xml
    at org.kitodo.config.ConfigProject.getDocType(ConfigProject.java:98)
    at org.kitodo.config.ConfigProject.getAdditionalFields(ConfigProject.java:189)
    at org.kitodo.production.process.NewspaperProcessesGenerator.initializeTitleGenerator(NewspaperProcessesGenerator.java:395)
    at org.kitodo.production.process.NewspaperProcessesGenerator.initialize(NewspaperProcessesGenerator.java:311)
    at org.kitodo.production.process.NewspaperProcessesGenerator.nextStep(NewspaperProcessesGenerator.java:279)
    at org.kitodo.production.helper.tasks.GeneratesNewspaperProcessesThread.run(GeneratesNewspaperProcessesThread.java:80)
```

**Comment by matthias-ronge** [comment]
Related: #3852

**Comment by matthias-ronge** [comment]
Simply fixing this is not possible because the classes that would require to be changed are deeply used by the broken rest interface. Cf. #3873

**Comment by matthias-ronge** [comment]
Should be fixed by pull request #3874
